app/containers/Questions/Content/Pagination/Pagination.tsx

```diff
@@ -55,152 +55,33 @@ const Pagination: React.FC<PaginationProps> = ({
               />
             ) : null,
           )}
-          {!lastPage && (
-            <StepButtonPagination
-              clickHandler={nextPage}
-              src={next}
-              scrollToTop={scrollToTop}
-              alt="next"
-            />
-          )}
         </div>
       </>
     );
   }
-  if (page <= 3 && totalPages > 5) {
+  if (totalPages > 4) {
```

Comment:
4 to constant

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1443/commits/6a91be129f1c647c9b813f5b83e914d5ad90010c